_posts/elements/2015-04-09-buttons.md

```diff
@@ -96,8 +96,8 @@ info: Buttons should be used for performing actions within the edX environment.
     </div>
 </section>
 
+<h3 class="hd-6 example-set-hd">Links</h3>
 <section class="example-set">
-    <h3 class="hd-6">Links</h3>
```

Comment:
Valid, accessible markup requires a heading to be the first child of a `section`. If we want these headings to be outside, we should convert the `sections` to `div`'s.